File: src/main/java/com/zerozero/ZerozeroApplication.java
package com.zerozero;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.openfeign.EnableFeignClients;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@EnableJpaAuditing
@EnableFeignClients
@SpringBootApplication
public class ZerozeroApplication {

    public static void main(String[] args) {
        SpringApplication.run(ZerozeroApplication.class, args);
    }

}

File: src/main/java/com/zerozero/ZerozeroApplication.kt
package com.zerozero

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.openfeign.EnableFeignClients
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@EnableJpaAuditing
@EnableFeignClients
@SpringBootApplication
class ZerozeroApplication {

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(ZerozeroApplication::class.java, *args)
        }
    }
}
